apps/social/src/components/layout/footer/socials/Socials.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactComponent as Twitter } from 'assets/images/socials/twitter.svg';
+import { ReactComponent as Github } from 'assets/images/socials/github.svg';
+import { ReactComponent as Discord } from 'assets/images/socials/discord.svg';
+import { ReactComponent as Medium } from 'assets/images/socials/medium.svg';
+import styles from './Socials.module.scss';
+
+const socials = [
```

Comment:
u can move it to constants

Reply:
I don't like this solution because these icons belong to the local component and shouldn't be where all the global values are.